### PR TITLE
chore: add targetCell values

### PR DESCRIPTION
**Pull request by justinkx**
https://developer.android.com/develop/ui/views/appwidgets/layouts#anatomy_determining_size

- [targetCellWidth](https://developer.android.com/reference/android/appwidget/AppWidgetProviderInfo#targetCellWidth) and [targetCellHeight](https://developer.android.com/reference/android/appwidget/AppWidgetProviderInfo#targetCellHeight) are more reliable and easy to configure compared to `minWidth or minHeight`. According to docs , **If defined, these attributes are used instead of minWidth or minHeight.**

### CODE REVIEW

**Comment by sAleksovski** [comment]
Please update the `CHANGELOG.md` file with

```md
## [Unreleased] (already there)

### Added

- `targetCellWidth` and `targetCellHeight` options in config plugin
```

and the docs wherever `android:minWidth="320dp"` is used, and add `targetCellWidth` = 5 and `targetCellHeight` = 2 values.

**Comment by justinkx** [comment]
> Also, did you try this on Android 11 or older?

In Android 11 and Older , cell width is not supported hence it will pick only minWidth and minHeight values.

File: app.plugin.ts [hunk 1]
```diff
@@ -375,6 +377,16 @@ function withWidgetProviderXml(
...
     android:minWidth="${widget.minWidth}"
     android:minHeight="${widget.minHeight}"
+${
+  widget.targetCellWidth
+    ? `    android:targetCellWidth="${widget.targetCellWidth}"`
+    : ''
+}
+${
+  widget.targetCellHeight
+    ? `    android:targetCellWidth="${widget.targetCellHeight}"`
```
Comment (sAleksovski) [inline app.plugin.ts:387]: `android:targetCellHeight` instead of `android:targetCellWidth`

**Comment by sAleksovski** [comment]
Thanks!

I just released `0.14.0` containing this feature.

**Pull request merged**